### DISCUSSION TITLE
[feature request] Grafana8 set exemplar=false for backwards compatibility

### DISCUSSION THREAD

**Discussion by 4orty** [grafana/grafana#37374] [Dashboards, Panels & Visualizations]
**What happened**:

After migration from 7 to 8, if i click any query in visualizations then automatically exemplar=true added in dashboard json Model even if i'm not using exemplar.

<img width="1181" alt="스크린샷 2021-07-29 오후 9 32 02" src="https://user-images.githubusercontent.com/49444468/127492371-0155b527-66e9-451c-9998-7dca3bd258ed.png">

We have some cases that exporting dashboard json Model in grafana 8 and import it to grafana 7 dashbaord, but due to exemplar=true option, it doesn't work in grafana 7 dashboard.

**What you expected to happen**:

If i don't use exemplar, i think the default value should be false.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.0.3
- Data source type & version: Prometheus (! without exemplar)
- OS Grafana is installed on: linux
- User OS & Browser: chrome
- Grafana plugins: 
- Others:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
